app/services/master_controller.py

```
@@ -770,68 +1047,183 @@ async def execute_5_phase_flow(
+            
+            # Get user's connected exchanges for coordinated arbitrage
+            from app.core.database import get_async_session
+            from app.models.exchange import ExchangeAccount
+            from sqlalchemy import select
+            
+            async with get_async_session() as db:
+                # Get user's exchange accounts
+                result = await db.execute(
+                    select(ExchangeAccount).where(
+                        ExchangeAccount.user_id == user_id,
+                        ExchangeAccount.is_active == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `ExchangeAccount.is_active:` for truth checks

Replace with `ExchangeAccount.is_active`

(E712)